changelog.d/20241105_162257_kurtmckee_reusable_tox_workflow.md

```diff
@@ -0,0 +1,4 @@
+### Other
+
+* Migrate to a reusable GitHub workflow.
+* Always recreate `-sdkmain` tox environments to ensure the SDK tarball is re-installed.
```

Comment:
Because the CLI changelog gets rendered onto `docs.globus.org`, I do not believe we should be recording these development-only updates in our changelog.

Reply:
I'll get that removed when I'm back on my work PC.

Reply:
Done. :partying_face: